packages/capabilities/src/types.ts

```diff
@@ -439,6 +442,99 @@ export interface UploadNotFound extends Ucanto.Failure {
 
 export type UploadGetFailure = UploadNotFound | Ucanto.Failure
 
+// HTTP
+export type HTTPPut = InferInvokedCapability<typeof HTTPCaps.put>
+
+// Blob
+export type Blob = InferInvokedCapability<typeof BlobCaps.blob>
+export type BlobAdd = InferInvokedCapability<typeof BlobCaps.add>
+export type ServiceBlob = InferInvokedCapability<typeof W3sBlobCaps.blob>
+export type BlobAllocate = InferInvokedCapability<typeof W3sBlobCaps.allocate>
+export type BlobAccept = InferInvokedCapability<typeof W3sBlobCaps.accept>
+
+export type BlobMultihash = Uint8Array
+export interface BlobModel {
+  digest: BlobMultihash
+  size: number
+}
+
+// Blob add
+export interface BlobAddSuccess {
+  site: {
+    'ucan/await': ['.out.ok.site', Link]
+  }
+}
```

Comment:
I suggest to define a type for await as I think we'll keep seeing it more

```suggestion
export interface UCANAwait<Selector extends string = string, Task extends unknown = unknown> {
  'ucan/await': [Selector, Link<Task>]
}

// Blob add
export interface BlobAddSuccess {
  site: UCANAwait<'.out.ok.site'>
}
```